comment/models/reactions.py

```diff
@@ -0,0 +1,90 @@
+from collections import namedtuple
+from typing import List
```

Comment:
Please don't use `typing`

Reply:
Any particular reason? They were introduced in 3.5 and we support versions later to their release. I like them personally but don't have an issue if you want them to be removed.

Reply:
It's just a personal openion, I don't like typing with python. However, in the future we may open an issue to refactor the code and add them.